Libs/RaptureWebServlet/src/main/java/rapture/server/web/servlet/UserBoundedScriptPageServlet.java

```diff
@@ -193,6 +246,35 @@ protected void runScriptWithUser(String script, Map<String, Object> parameterMap
         resp.flushBuffer();
     }
 
+    /**
+     * Helper method to extract the URI from HttpServletRequest
+     * <p>
+     * @param  request  The actual script/endpoint to execute  
+     * @return          URI of script as a String e.g. script://<repo>/endpoint 
+     */
+    private String getScriptUrlFromHttpRequest(HttpServletRequest request){
+        String scriptURL = "";
```

Comment:
declare this when its first used on line 265.  no need to initialize to empty string here.